pom.xml

```diff
@@ -4,28 +4,84 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.cloud.tools.login</groupId>
-  <artifactId>google-login-common</artifactId>
-  <version>0.0.0-SNAPSHOT</version>
+  <artifactId>plugins-login-common</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <name>Login Support for Google Cloud Tools</name>
+  <description>
+    A library providing a common framework for Google login support for Google Cloud Tools.
+  </description>
+  <url>https://github.com/GoogleCloudPlatform/ide-login</url>
+
+  <organization>
+    <name>Google Inc.</name>
+    <url>https://www.google.com</url>
```

Comment:
I wonder if this should be Google Cloud Platform and https://cloud.google.com/ rather than Google Inc. No idea if this matters.